conf/modules.config

```diff
@@ -12,23 +12,35 @@
 
 process {
 
-    publishDir = [
-        path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-    ]
+    if(params.publish_dir_all_tools) {
+        publishDir = [
+            path: { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+    }
 
-    withName: FASTQC {
+    withName: 'FASTQC' {
         ext.args = '--quiet'
-    }
+        memory = '12.GB'
```

Comment:
Perhaps instead of being harcoded this could be written in `base.config` like here: https://github.com/nf-core/sarek/blob/5cc30494a6b8e7e53be64d308b582190ca7d2585/conf/base.config#L73-L76

Reply:
I've removed the line